### DISCUSSION TITLE
Standalone installation Ubuntu/Debian Server

### DISCUSSION THREAD

**Discussion by Urajmal** [DefGuard/defguard#685] [Q&A] [answered]
Dear Developers!

As I did really get a lot of positive feedback about defguard, I would like to know if there is a way to install standalone version directly on Linux Server (Ubuntu / Debian)? 

I am currently using ZTnet, which is fantastic ZeroTier instance and has a clear straight way to install standalone version (https://ztnet.network/installation/linux). 

While I am not really keen on Docker and Docker Compose, since I tried to install them and was unable to set up nor the installation line did not execute OK, I am interested in regular standalone Linux Installation. 

Thank you!

**Comment by teon** [comment]
Hi there!

Please watch this issue: https://github.com/DefGuard/docs/issues/29

When it's finished on our official documentation there will be a chapter describing standalone package based install.

Ps. Could you tell us where did you hear about defguard? 🙏

**Comment by Urajmal** [comment]
Thank you for fast answer. I am eagerly waiting for the standalone package for Ubuntu LTS Server od Debian.

I saw the mention of Defguard here: 
https://www.reddit.com/r/selfhosted/comments/18nrbnf/wireguard_admin_interface_ui_that_is_actively/

**Reply by bmullan** [reply]
Me too

**Comment by Urajmal** [comment]
Thank you for the answer! 

Eagerly waiting for installation package.

**Comment by teon** [comment, marked as answer]
https://defguard.gitbook.io/defguard/admin-and-features/setting-up-your-instance/standalone-package-based-installation
